modules/users/data/data.yaml

```diff
@@ -68,7 +68,14 @@ groups:
                  'ALL = (ALL) NOPASSWD: /usr/sbin/service mariadb status',
                  'ALL = (ALL) NOPASSWD: /usr/bin/puppet *',
                  'ALL = (ALL) NOPASSWD: /bin/journalctl *']
-
+  mediawiki-test-deploy-users:
+    gid: 2009
+    description: ability to deploy changes on the MediaWiki test server
+    members: []
+    privileges: ['ALL = (ALL) NOPASSWD: /bin/nano /srv/mediawiki/*',
```

Comment:
does this group _really_ need this? I think they shouldn't be able to edit or remove files on /srv/mediawiki/* and should just edit mediawiki-staging and use `mwdeploy --file` to deploy changes